components/instagram_business/actions/create-post/create-post.mjs

```diff
@@ -0,0 +1,61 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  key: "instagram_business-create-post",
+  name: "Create Post",
+  description: "Post a single image, video, or reel. [See the documentation](https://developers.facebook.com/docs/instagram-api/guides/content-publishing#single-media-posts)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    mediaType: {
+      type: "string",
+      label: "Media Type",
+      description: "The type of media to post",
+      options: [
+        "image",
```

Comment:
You could use an object with { label, value }, to prevent the IF on the line 41, also, I would move these options to a constants file.